src/pudl/metadata/fields.py

```diff
@@ -2215,6 +2215,26 @@
         "type": "boolean",
         "description": "indicates if the boiler is a heat recovery steam generator (HRSG).",
     },
+    "in_rate_base": {
+        "type": "boolean",
+        "description": (
+            "Whether or not a record from the detailed FERC1 accounting tables should "
+            "be considered allowable in a utility's rate base based on utility "
+            "accounting standards. "
+            "This flag was mannually compiled by RMI utility accounting experts "
+            "based on the xbrl_factoid and sometimes varies based on the utility_type "
+            "plant_status or plant_function as well."
```

Comment:
nit: add commas between the items in this list. Also, I think you can omit the "as well" at the end as the "or" implies that it's either or.

(applicable to all definitions where this is the case).

Reply:
this was actually the only one that could vary by three dimensions!